app/src/python/tests/test_user_scripts.py

```diff
@@ -33,8 +33,8 @@ def test_gamma_calc(temp_workingdir):
     runpy.run_path(scriptpath)
 
 
-# def test_mu_density_diff(temp_workingdir):
-#     os.chdir(temp_workingdir)
-#     scriptpath = os.path.join(USER_SCRIPTS, 'mu-density-diff.py')
+def test_mu_density_diff(temp_workingdir):
+    os.chdir(temp_workingdir)
```

Comment:
Now running a "test", where two of the user-scripts are run to see if they don't raise an error.